webkitgtk.yml

```diff
@@ -0,0 +1,163 @@
+# And now, WebkitGTK and its deps *specifically.*
+# Big thanks to Telegram Desktop Webview's manifest maintainers for this.
+# Because I took pretty much the whole stupid thing and refactored it into its
+# own yml file. I then added many extra parts from all over the Flathub org.
+# Anyone who needs to include webkitgtk, *please* feel free to use this.
+# This is from https://github.com/flathub/org.geany.Geany, and thanks to them we don't need to deal with this pain again.
+
+name: webkit2gtk-4.0
```

Comment:
You added too many dependencies, only this is needed https://github.com/flathub/in.cinny.Cinny/blob/263ff5d11ee96368a36e24e5adffc742f1b93f2d/in.cinny.Cinny.yml#L32-L64 and put this in the main manifest.

Delete highway, libavif, woff2, libjxl, libwpe, wpebackend-fdo, some of these are in the runtime, some are needed.